### ISSUE TITLE
Add ComputeAutoscaler support 

### ISSUE THREAD

**Issue by caieo** [GoogleCloudPlatform/k8s-config-connector#549]
**Describe the feature or resource**
* [ComputeAutoscaler](https://cloud.google.com/compute/docs/reference/rest/v1/autoscalers)

**Comment by DWSR** [comment]
Yeah, I just discovered that this is missing. It makes MIGs pretty painful to use.

**Comment by ldanielmadariaga** [comment]
This item has been open for >12 months. As part of scheduled repository maintenance, this Issue/PR will be closed. If this is still a critical ask, please re-open and work with your Google Cloud Account team to prioritize it with GCP KCC Product Management.
